packit_service/worker/handlers/abstract.py

```diff
@@ -210,5 +212,26 @@ def project(self) -> Optional[GitProject]:
             self._project = self.config.get_project(url=self.data.project_url)
         return self._project
 
-    def run(self) -> HandlerResults:
+    def run_job(self):
+        """
+        If pre-check succeeds, run the job for the specific handler.
+        :return: Dict [str, TaskResults]
+        """
+        job_type = self.job_config.type or self.type
+        logger.debug(f"Running handler {str(self)} for {job_type}")
+        job_results: Dict[str, TaskResults] = {}
+        if self.pre_check():
+            current_time = datetime.now().strftime(DATETIME_FORMAT)
+            result_key = f"{job_type.value}-{current_time}"
+            job_results[result_key] = self.run_n_clean()
+            logger.debug("Job finished!")
+
+            for v in job_results.values():
+                if not (v and v["success"]):
+                    logger.warning(job_results)
+                    logger.error(v["details"]["msg"])
```

Comment:
It's maybe copied from another place, but I would prefer longer names..;)

```suggestion
            for result in job_results.values():
                if not (result and result["success"]):
                    logger.warning(job_results)
                    logger.error(result["details"]["msg"])
```

---

Do we really want to log the whole `job_results` before each nonsuccessful result?

```python
logger.warning(job_results)
```

Reply:
yes, this was copied, but changed now